src/main/resources/db/migration/postgresql/V2.15.0.20240515220400_atlas_global_share_permission.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO ${ohdsiSchema}.sec_permission
+(value, description)
+values
+('cohortdefinition:global:share:put', 'special permission (intended for admins) that allows the user to share any cohort with a "global reader role"')
```

Comment:
todo- change to "artifact"

```suggestion
('artifact:global:share:put', 'special permission (intended for admins) that allows the user to share any cohort with a "global reader role"')
```